Makefile

```diff
@@ -154,7 +154,7 @@ ifeq ("$(TRAVIS_COVERAGE)", "1")
 			|| { $(FAILPOINT_DISABLE); exit 1; }
 else
 	@echo "Running in native mode."
-	@export log_level=error; export TZ='Asia/Shanghai'; \
+	@export log_level=fatal; export TZ='Asia/Shanghai'; \
```

Comment:
Why make this change?

Reply:
The error level is a bit noisy for the developer.